pyproject.toml

```diff
@@ -122,3 +123,46 @@ showcontent = true
 directory = "removal"
 name = "Removals"
 showcontent = true
+
+[tool.bumpversion]
+current_version = "0.1.0-alpha.0"
```

Comment:
I think this is why I got `v0.1.1` above. Should be 4.1.1

Reply:
+1, this needs to be changed to `4.1.1` before merge.